po/es.po

```diff
@@ -11398,15 +11400,15 @@ msgstr ""
 #: ../src/iop/vignette.c:1193
 msgid "enable to have the ratio automatically follow the image size"
 msgstr ""
-"activar para que la proporción siga el tamaño de imagen automáticamente"
+"Activar para que la proporción siga automáticamente el tamaño de imagen"
 
 #: ../src/iop/vignette.c:1194
 msgid "width-to-height ratio"
-msgstr "proporción ancho/alto"
+msgstr "Factor de proporción entre la altura y la anchura"
 
 #: ../src/iop/vignette.c:1195
 msgid "add some level of random noise to prevent banding"
-msgstr "agregar algun nivel de ruido aleatorio para prevenir bandas"
+msgstr "Añadir cierto nivel aleatorio de ruido para prevenir bandas"
```

Comment:
Current translated sentence means "add some random level of noise...".
I guess, it's more right to say "Añadir cierto nivel de **ruido aleatorio** para prevenir bandas"

Reply:
Thanks!